pkg/jiraservicedesk/client/client_test.go

```diff
@@ -0,0 +1,78 @@
+package client
+
+import (
+	"errors"
+	"testing"
+
+	"github.com/nbio/st"
+	"gopkg.in/h2non/gock.v1"
+)
+
+var endpoint = "177"
+var client = NewClient("", "https://mock.atlassian.net/", "")
+var jira_url = "https://mock.atlassian.net/rest/api/3"
```

Comment:
For slug(the part after base url) use https://github.com/stakater/jira-service-desk-operator/blob/master/pkg/jiraservicedesk/client/client_project.go#L14 instead of hard-coding it here. 

In that case we'll just have a single place to update the path if it's required.